elfloader-tool/src/arch-arm/plat-zynq7000/sys_fputc.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2014, NICTA
```

Comment:
The NICTA license header is no longer used. Currently we add new files with

```c
/*
 * Copyright 2016, Data61
 * Commonwealth Scientific and Industrial Research Organisation (CSIRO)
 * ABN 41 687 119 230.
 *
 * This software may be distributed and modified according to the terms of
 * the GNU General Public License version 2. Note that NO WARRANTY is provided.
 * See "LICENSE_GPLv2.txt" for details.
 *
 * @TAG(D61_GPL)
 */
```

If you would be so kind as to change that